### PR TITLE
[#9760] Allow passing range as a positional argument to RangeHelp#range_with_surrounding_space

### PR DESCRIPTION
**Pull request by pirj**
Related to #9760
Prequel to #10729 and #10727

-----------------

Before submitting the PR make sure the following are checked:

* [x] The PR relates to *only* one subject with a clear title and description in grammatically correct, complete sentences.
* [x] Wrote [good commit messages][1].
* [x] Commit message starts with `[Fix #issue-number]` (if the related issue exists).
* [x] Feature branch is up-to-date with `master` (if not - rebase it).
* [x] Squashed related commits together.
* [x] Added tests.
* [x] Ran `bundle exec rake default`. It executes all tests and runs RuboCop on its own code.
* [x] Added an entry (file) to the [changelog folder](https://github.com/rubocop/rubocop/blob/master/changelog/) named `{change_type}_{change_description}.md` if the new code introduces user-observable changes. See [changelog entry format](https://github.com/rubocop/rubocop/blob/master/CONTRIBUTING.md#changelog-entry-format) for details.

[1]: https://chris.beams.io/posts/git-commit/

### CODE REVIEW

**Comment by bbatsov** [comment]
Looks good to me. Thanks!

File: lib/rubocop/cop/mixin/range_help.rb [hunk 1]
```diff
@@ -51,9 +51,13 @@ def range_with_surrounding_comma(range, side = :both)
         Parser::Source::Range.new(buffer, begin_pos, end_pos)
       end
 
-      def range_with_surrounding_space(range:, side: :both,
-                                       newlines: true, whitespace: false,
-                                       continuations: false)
+      NOT_GIVEN = Module.new
```
Comment (bquorning) [inline lib/rubocop/cop/mixin/range_help.rb:54]: Just curious – why is `NOT_GIVEN` a module and not e.g. a symbol?

Comment (pirj) [reply lib/rubocop/cop/mixin/range_help.rb:54]: `Module.new == Module.new` evaluates to `false`. Chances of someone passing a `:not_given` positional argument are nearing zero, but anyway.

Comment (bquorning) [reply lib/rubocop/cop/mixin/range_help.rb:54]: Got it, thanks.

**Pull request merged**